### PR TITLE
Find a way to track migration initiated from MFE

### PR DESCRIPTION
**Pull request by ramyakrishnai**
## Proposed changes
JIra:  https://jira.newfold.com/browse/PRESS0-1764

## Type of Change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [ ] I have read the [CONTRIBUTING](https://github.com/bluehost/.github/blob/master/.github/contributing.md) doc
- [ ] I have viewed my change in a web-browser
- [ ] Linting and tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: src/OnboardingSPA/components/NewfoldInterfaceSkeleton/SiteBuild/index.js [hunk 1]
```diff
@@ -403,6 +405,7 @@ const SiteBuild = () => {
 
 	// Track Migration Initated Event in the Migration Step.
 	useEffect( () => {
+		fetchWPSettings().then( ( res ) => ( setIsMfeMigrationInitiated( res.nfd_migrate_site ) ) );
```
Comment (officiallygod) [inline src/OnboardingSPA/components/NewfoldInterfaceSkeleton/SiteBuild/index.js:408]: Can we not emit the use of a state variable `isMfeMigrationInitiated` and directly call trackInstaWpMigrationEvent with a value as a param and use that to determine if `variable === true ? ACTION_MFE_MIGRATION_INITIATED: ACTION_MIGRATION_INITIATED ` ?

I think using state variables to do one-time update is an overhead as state variables may take time to reflect.

**Review by officiallygod — APPROVED** [review]
Looks Good ✅

**Comment by officiallygod** [comment]
PRESS0-1764

**Comment by ramyakrishnai** [comment]
[PRESS0-1764](https://jira.newfold.com/browse/PRESS0-1764)

**Pull request merged**